Polkadot status up to Decoded 2023 (#4998)
w3f/polkadot-wiki: docs/general/polkadot-vision.md

---
id: polkadot-vision
title: Polkadot Vision
sidebar_label: Polkadot Vision
description: Polkadot's Vision revealed by Gavin Wood at Decoded 2023.
keywords: [polkadot vison, polkadot]
slug: ../polkadot-vision
---

## A Multi-chain Future

Since the release of Bitcoin in 2009, blockchain projects increased exponentially to the order of
tens of thousands. Different projects have different value propositions, suggesting that the future
will likely be multi-chain and that inter-chain communication will be crucial to establish

> collaborations and leveraging each other strengths.

Reviewer: ```suggestion
Since the release of Bitcoin in 2009, blockchain projects increased exponentially to the order of
tens of thousands. Different projects have different value propositions, suggesting that the future
will be multi-chain and that inter-chain communication will be crucial to establish
collaborations and leveraging each other strengths.
```

## Polkadot 1.0

Polkadot 1.0 reflects the status of Polkadot in 2023 at time of the release of the Polkadot runtime
v1.0. This sections focuses on Polkadot 1.0 and some philosophical digressions about network
resilience and blockspace.

Polkadot is a Layer-0 blockchain that brings to the multi-chain vision the following innovative

> concepts and initiatives:

Reviewer: ```suggestion
Polkadot is a Layer-0 blockchain that brings to the multi-chain vision the following innovations and initiatives:
```

- **Application-specific Layer-1 (L1) blockchains (or parachains).** Polkadot is a sharded network
  where data are processed in parallel within each shard. Polkadot shards can be heterogenous (i.e.
  they do not need to be the exact copy of the original shards, like in Ethereum shards). This

>   allows to build L1 chains designed explicitly around their application and value proposition.

Reviewer: ```suggestion
- **Application-specific Layer-1 (L1) blockchains (or parachains).** Polkadot is a sharded network
  where transactions are processed parallelly within each shard. Polkadot shards can be heterogenous (i.e.
  they do not need the same state transition function as in the proposed Ethereum sharding architecture). This
  allows to build L1 chains designed explicitly around their application and value proposition.
```

- **Shared security and financial scalability of L1 chains**. Any L1 chain attached to a Polkadot
  core (and any other L2 solution to scale that L1) will benefit from Polkadot shared security
  model. This means the Polkadot Nominated-Proof-of-Stake (NPoS) mechanism will secure L1 and L2

>   chains without them worrying about having enough proof-of-stake.

Reviewer: L2 security assumptions differ from parachain security assumptions. By design. all the parachains enjoy the same levels of security guarantees provided by Polkadot. The L2 solutions developed on top of L1s do not, as they do not adhere to any particular standard.

```suggestion
- **Shared security and financial scalability of L1 chains**. Any L1 chain attached to a Polkadot
  core can benefit from Polkadot shared security
  model. This means the Polkadot Nominated-Proof-of-Stake (NPoS) mechanism along with its consensus mechanism, will secure L1
  chains out-of-the-box without having to bootstrap security on their own.
```

> - **Secure interoperability.** Any L1 chain attached to Polkadot will benefit from Polkadot's native

Reviewer: ```suggestion
- **Secure interoperability.** Any L1 chain attached to Polkadot (as well as L2 chains built on top of them) can benefit from Polkadot's native
```

  interoperability, and will thus be able to communicate and exchange value and information with
  other parachains.
- **Truly resilient infrastructure.** This is achieved by keeping the network decentralized without
  compromising scalability and throughput, and through on-chain treasury funds that can be accessed
  through governance referendum. Those funds guarantee constant sponsorship for events, initiatives,
  educational material, software development, etc., without any central entity being able to

>   interfere.

Reviewer: ```suggestion
  education, software development, etc.
```

> - **Fast deployment of L1 chains.** This is achieved through the modular and flexible

Reviewer: ```suggestion
- **Fast development and deployment of L1 chains.** This is achieved through the modular and flexible
```

  [Polkadot SDK Substrate](./glossary.md#substrate).
- **Fostering next-gen of Web3 core developers.** This is achieved through different initiatives
  such as:
  - [The Polkadot Blockchain Academy](https://polkadot.network/development/academy/)
  - [Substrate Builders Program](./builders-program.md)
  - [Polkadot Developer Heroes Program](./dev-heroes.md)
  - [Edx Courses](https://www.edx.org/school/web3x)
  - Rust and Substrate Courses (coming soon)

### Polkadot's Representation

> The Polkadot realy-chain has been represented as a central ring surrounded by multiple parachains

Reviewer: ```suggestion
The Polkadot realy-chain has been represented as a relay chain surrounded by multiple parachains
```

attached to it. Parachains can communicate with each other using a generalized
[cross-chain messaging format (XCM)](./learn-xcm-index) allowing messages to be delivered across
chains having different logic.

> ### Polkadot's Additional Tasks

Reviewer: ```suggestion
### Polkadot's Additional Functionalities
```

The Polkadot relay-chain also manges [crowdloans](../learn/learn-crowdloans.md),
[auctions](../learn/learn-auction.md), [staking](../learn/learn-staking.md),
[accounts](./learn-accounts-index), [balances](../learn/learn-balance-transfers.md), and
[governance](../learn/learn-polkadot-opengov.md). Parachain slots or cores are leased in 6-month
chunks for a maximum of two years, and crowdloans allow users to trustlessly loan funds to teams for
lease deposits in exchange for pre-sale tokens. There is no other way you could use Polkadot 1.0.

### Decentralization as a Tool

Decentralization is a crucial aspect when building a blockchain, but there must be a trade-off

> between:

Reviewer: ```suggestion
Decentralization is a crucial aspect of blockchain networks, but there is a trade-off
between:
```

- having an over-decentralized network that struggles to reach consensus and consumes a lot of

>   energy to stay online, and

Reviewer: ```suggestion
  energy to operate, and
```

- having an under-decentralized network that reaches consensus fast at the expense of being

>   centralized and trivial to attack.

Reviewer: ```suggestion
- having a network that reaches consensus fast at the expense of being
  centralized, making it trivial to manipulate or attack.
```

> Ideally, a network should be decentralized enough to make it practically impossible for someone to

Reviewer: ```suggestion
Ideally, a network should be decentralized "enough" to make it practically impossible for someone to
```

> stop all nodes and take the network down. So, decentralization is a tool while the goal is

Reviewer: ```suggestion
exert manipulative or malicious influence on the network. So, decentralization is a tool while the goal is
```

resilience, which is achieved by additionally providing on-chain treasury and governance mechanism

> allowing continuous financial incentives for the network's participants without intermediaries or

Reviewer: ```suggestion
allowing continuous incentives for the network's participants without relying on intermediaries or
```

> centralized entities being able to stop the network's progress.

Reviewer: ```suggestion
centralized entities.
```

> Currently, Polkadot 1.0 achieve resilience through the followings:

Reviewer: ```suggestion
Currently, Polkadot 1.0 achieve resilience through the following strategies:
```

> - The stake per validator is maximized and evenly distributed across validators.

Reviewer: ```suggestion
- Nominated Proof of Staking (NPoS) where the stake per validator is maximized and evenly distributed across validators.
```
